app-emulation/genymotion-bin/genymotion-bin-3.3.2.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 desktop pax-utils
+
+MY_PN="${PN/-bin}"
+MY_P="${MY_PN}-${PV}"
+BIN_ARCHIVE="${MY_P}-linux_x64.bin"
+
+DESCRIPTION="Complete set of tools that provide a virtual environment for Android"
+HOMEPAGE="https://genymotion.com"
+SRC_URI="${BIN_ARCHIVE}"
+
+LICENSE="genymotion"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+RDEPEND="app-emulation/virtualbox
+	|| (
+		dev-libs/openssl-compat:1.1.0
+		=dev-libs/openssl-1.1*:0
+	)
+	dev-libs/glib:2
+	=dev-libs/hiredis-1.0*
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/libpulse
+	media-libs/gst-plugins-base
+	sys-libs/zlib
+	virtual/opengl
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXi
+	x11-libs/libxkbcommon
+	x11-libs/libXmu
+	x11-libs/libXrender
+	x11-misc/xdg-utils
```

Comment:
Then:
```
 * VDB: detected possibly incorrect RDEPEND (app-emulation/genymotion-bin-3.3.2)
 *                             > media-libs/alsa-lib
 * media-libs/gst-plugins-base | media-libs/gst-plugins-base:1.0
 *                             > media-libs/gstreamer:1.0
 *                             > sys-apps/dbus
 *                             > x11-libs/libxcb
 * x11-libs/libXrender         < 
 *                             > x11-libs/xcb-util
 *                             > x11-libs/xcb-util-image
 *                             > x11-libs/xcb-util-keysyms
 *                             > x11-libs/xcb-util-renderutil
 *                             > x11-libs/xcb-util-wm
```
You'll have to add the ones on right, the ones in left can be removed / update media-libs/gst-plugins-base to pin on the exact slot as suggested.